Seems x86 builds are bugged:

```

======================================================================
ERROR: issue_1912_poc.jpg_test (test_regression_allfiles.TestAllFiles.issue_1912_poc.jpg_test)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "D:\a\exiv2\exiv2\tests\regression_tests\test_regression_allfiles.py", line 191, in test_func
    BT.reportTest(os.path.basename(filename), out)
  File "D:\a\exiv2\exiv2\tests\bash_tests\utils.py", line 567, in reportTest
    raise RuntimeError('\n' + log.to_str())
RuntimeError: 
Error:  The output of the testcase mismatch the reference
[INFO] The output has been saved to file D:\a\exiv2\exiv2\test\tmp\issue_1912_poc.jpg.out
[INFO] simply_diff:
D:\a\exiv2\exiv2\test/data\test_reference_files\issue_1912_poc.jpg.out: 53 lines
D:\a\exiv2\exiv2\test\tmp\issue_1912_poc.jpg.out: 53 lines
The first mismatch is in line 49:
< Exif.GPSInfo.GPSTimeStamp                    Rational    3  1131375984/1869505902 1950679091/1 18/1  05:20:00
> Exif.GPSInfo.GPSTimeStamp                    Rational    3  1131375984/1869505902 1950679091/1 18/1  05:20:18
```

--- a/.github/workflows/on_PR_windows_matrix.yml
+++ b/.github/workflows/on_PR_windows_matrix.yml
@@ -33,19 +33,9 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: Set up Visual Studio shell
-        uses: egor-tensin/vs-shell@v2
+        uses: ilammy/msvc-dev-cmd@v1
         with:
           arch: ${{matrix.platform}}
-
-      - name: Set up Ninja
-        uses: ashutoshvarma/setup-ninja@master
-        with:
-          version: 1.10.0
-
-      - name: Set up Python
-        uses: actions/setup-python@v4
-        with:
-          python-version: 3.7
 
       - name: Restore Conan cache
         uses: actions/cache@v3

--- a/.github/workflows/on_PR_windows_matrix.yml
+++ b/.github/workflows/on_PR_windows_matrix.yml
@@ -76,7 +76,6 @@ jobs:
           cmake --install build
 
       - name: Test
-        if: ${{matrix.platform == 'x64'}}
         run: |
           ctest --test-dir build --output-on-failure
 

--- a/.github/workflows/on_push_BasicWinLinMac.yml
+++ b/.github/workflows/on_push_BasicWinLinMac.yml
@@ -22,20 +22,10 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Setup Ninja
-        uses: ashutoshvarma/setup-ninja@master
-        with:
-          version: 1.10.0
-
       - name: Set up Visual Studio shell
-        uses: egor-tensin/vs-shell@v2
+        uses: ilammy/msvc-dev-cmd@v1
         with:
           arch: x64
-
-      - name: Set up Python
-        uses: actions/setup-python@v4
-        with:
-          python-version: 3.7
 
       - name: Restore conan cache
         uses: actions/cache@v3

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -98,17 +98,7 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: Set up Visual Studio shell
-        uses: egor-tensin/vs-shell@v2
-
-      - name: Setup Ninja
-        uses: ashutoshvarma/setup-ninja@master
-        with:
-          version: 1.10.0
-
-      - name: Set up Python
-        uses: actions/setup-python@v4
-        with:
-          python-version: 3.7
+        uses: ilammy/msvc-dev-cmd@v1
 
       - name: Install doxygen
         run: |

